script/Ciel/Day 5/05_01_CIEL05_4C.txt

```diff
@@ -1,62 +1,65 @@
 [sha:1955c118575165462cf76559961f5e925f825ccf]{
 -- Page 10025, Offset 12935.
 -- 「なんだ―――結局、先輩だけで良かったんじゃないか」
-"Huh―――In the end, it would've been fine even if you were alone, Senpai."
+"Geez... I guess senpai would have been fine on her own after all."
 }
 [sha:61f5d56c9b22eadafbe8614c94920b497c109836]{
 -- Page 10025, Offset 12936.
 -- 仰向けに寝返りをうちながら、悔しいような、誇らしいような、そんな感想がこぼれていた。
-As I roll over on my back, I let my emotions out, part frustration, part pride.
+As I roll over on my back, I vent my feelings of both frustration and pride.
 }
 [sha:eab7c89dcebdf5103504217b41272dd8a6eb0628]{
 -- Page 10025, Offset 12937.
 -- 心肺機能が喜びの悲鳴をあげる。
-My heart and lungs scream with relief.
+My heart and lungs cry out in relief.
 }
 [sha:08d5adc313866e2b4bbccc3e667304c4c46b1424]{
 -- Page 10025, Offset 12938.
 -- 長く止められていた呼吸を、ようやく再開させられた。
-I finally resume breathing, after having stopped a long time ago.
+After a long time spent holding my breath, I finally begin breathing again.
 }
 [sha:3f5d984bbb1b21bd1d5c27df0c400994b04b53c8]{
 -- Page 767, Offset 12939.
 -- 体はもう動きもしない。
-My body can't move anymore.
+I can't move my body anymore.
 }
 [sha:1b06c3653ef2721ccc6496108b9cd62f2b5fc6c2]{
 -- Page 767, Offset 12940.
 -- ナイフを握っているかどうかも曖昧だ。
-It's not even clear to me whether I'm still gripping my knife.
+I can't even tell whether I'm still gripping my knife.
 }
 [sha:41435476a538bf8d700b52d2a7616640d0d1fdd4]{
 -- Page 767, Offset 12941.
 -- 確かなものは頭蓋の中の痛みと、気を緩めた途端、闇に落ちそうな意識だけ。
-What I do feel is the pain inside my head, and the sense of falling into the darkness, as soon as I relax.
+All I can feel for certain is the pain inside my head,
+ and the sense that as soon as I relax,
+ my consciousness will plummet into darkness.
 }
 [sha:6c79eba496d1ff9982ffa2dd80fa8d7374f40587]{
 -- Page 768, Offset 12942.
 -- 左肩に刺さった氷の痛みも、
-Like I'm under anesthesia, I don't feel the pain of the ice prickling my left shoulder.
+I don't feel any pain where the ice pierced my shoulder.
 }
 [sha:2f3d97b3394492f6b9de1e41c96171a37100763e]{
 -- Page 768, Offset 12943.
 -- 切り刻まれた手足からの出血も、今は麻酔にかかったように感じない。
-Nor the bleeding of my mangled limbs.
+I can't sense the bleeding of my mangled limbs either.
+ It's as though I've been put under anesthesia.
 }
 [sha:be38f3de691262f3c60b3c808e80d02f5e63b157]{
 -- Page 768, Offset 12944.
 -- 頭痛を薄める為、なんとか眼鏡をかけたところで、完全に電池が切れた。
-To alleviate my headache, I manage to put on my glasses, but it's no use, it's like I've been completely cut off from power.
+I somehow manage to put my glasses back on in order to alleviate my headache, but with that I'm completely and utterly spent.
 }
 [sha:327af894e9a67f3229af04e4aa17c172a8c3515c]{
 -- Page 769, Offset 12945.
 -- 「遠野くん……！　しっかり、生きていますか……!?」
-"Tohno-kun...! Hold tight, are you still alive...!?"
+"Tohno-kun...! Are you okay, are you still alive...!?"
```

Comment:
I see what you're trying to do here but if you want to split it out, it needs its own punctuation imo
> Are you okay?! Are you still alive...!?